distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -850,8 +860,8 @@ Text.repeat self count=1 =
          "Hello World!".take (By_Index [Range 0 3, 6, Range 6 12 2]) == "HelWWrd"
          "Hello World!".take (Sample 3 seed=42) == "l d"
 Text.take : (Text_Sub_Range | Index_Sub_Range | Range | Integer) -> Text ! Index_Out_Of_Bounds_Error
-Text.take self range=(First 1) =
-    ranges = Text_Sub_Range_Module.find_codepoint_ranges self range
+Text.take self range=(Index_Sub_Range.First 1) =
+    ranges = Codepoint_Ranges.find self range
```

Comment:
With this new layout of this method, I'm thinking if `Codepoint_Ranges.resolve` will not sound better.

Reply:
Will rename.